common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -381,12 +388,85 @@ def test_partially_complete_notifications(self):
         self.problem_page.wait_partial_notification()
 
 
-class AnnotationProblemTypeTest(ProblemTypeTestBase, ProblemTypeTestMixin):
+@ddt.ddt
+class ProblemNeverShowCorrectnessMixin(object):
+    """
+    Tests the effect of adding `show_correctness: never` to the sequence metadata
+    for subclasses of ProblemTypeTestMixin.
+    """
+    sequential_metadata = {'show_correctness': 'never'}
+
+    @attr(shard=7)
+    @ddt.data('correct', 'incorrect', 'partially-correct')
+    def test_answer_says_submitted(self, correctness):
+        """
+        Scenario: I can answer a problem <Correctness>ly
+        Given External graders respond "<Correctness>""
+        And I am viewing a "<ProblemType>" problem
+        in a subsection with show_correctness set to "never"
+        Then I should see a score of "N point(s) possible (ungraded, results hidden)"
+        When I answer a "<ProblemType>" problem "<Correctness>ly"
+        Then my "<ProblemType>" answer is marked "<Correctness>"
+        And The "<ProblemType>" problem displays only a "submitted"
+        notification.
+        And I should see a score of "N point(s) possible (ungraded, results hidden)"
```

Comment:
It is, but it indicates that the problem progress area is unchanged after submitting an answer.